index.html

```diff
@@ -20,8 +20,8 @@
                 </ul>
             </nav>
             <div id="header-heading">
-                Hey there. I’m Clifford.<br />
-                I’m a software developer
+                Hey there. I’m Clifford.
+                <span>I’m a software developer</span>
```

Comment:
- [OPTIONAL] Your desktop version is looking good as suggested by the previous reviewer. However, your mobile version seems to be broken. Maybe it's because of the implemented changes. Please see the test result.

Demo Test |
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/171996861-5c2ff522-0570-48a9-985e-9b6b37845670.gif)